isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile

```diff
@@ -0,0 +1,18 @@
+# Test for new format of Hayabusa2 fits images released 2019-2020
+#
+# @history 2020-03-29 Kristin Berry - Added along with support for new format
+#
+APPNAME = hyb2onc2isis
+
+include $(ISISROOT)/make/isismake.tsts
+
+commands:
+	$(APPNAME) from=$(INPUT)/hyb2_onc_20180710_060508_tvf_l2a.fit  \
+	           to=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub   \
+	           >& /dev/null;
+	catlab     from=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub \
+	           to=$(OUTPUT)/labels.pvl                             \
+	           >& /dev/null;
+	catoriglab from=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub \
+	           to=$(OUTPUT)/origLab.pvl                            \
+	           >& /dev/null;
```

Comment:
Is the test data for this already checked in?

Reply:
No.

Reply:
For new tests, it's okay to check in the test data early as it won't conflict with other PRs or dev.

Reply:
Forgot about that, good point.